extensions/attributegen/config.proto

```diff
@@ -0,0 +1,113 @@
+/* Copyright 2020 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+package istio.attributegen;
+
+// Configuration to generate attributes based on existing attributes.
```

Comment:
Nit: This is a bit difficult to understand, suggestion: 

Configuration to generate synthetic attributes based on attributes of the proxied traffic

Something like this might be more clear.